src/dbt_mcp/config/config.py

```diff
@@ -87,8 +89,11 @@ def actual_prod_environment_id(self) -> int | None:
 
     @field_validator("disable_tools", mode="before")
     @classmethod
-    def parse_disable_tools(cls, value: str) -> list[str]:
-        return [tool.strip() for tool in value.split(",") if tool.strip()]
+    def parse_disable_tools(cls, env_var: str | list) -> list[ToolName]:
```

Comment:
Why do we have a `str | list` as input here? As we have set `NoDecode` for `disable_tools` it will always be considered a `str` (or None maybe).

Because of `NoDecode` we can then set our own parsing